editor/configs/peopleColumns.tsx

```diff
@@ -43,6 +43,18 @@ export const ALL_PEOPLE_COLUMNS = {
       labelColumnName: "label",
     },
   },
+  occpnt_pos: {
+    path: "occpnt_pos.label",
+    label: "Occupant Position",
+    editable: false,
```

Comment:
i just messaged Xavier to confirm he doesn't want to be able to edit this 👍

Reply:
@roseeichelmann would you make this editable? Xavier would like to be able to edit this.

Reply:
so just to make sure - xavier does not need to edit this?

Reply:
sorry for the confusion. Xavier **does** need to edit. Please make this editable.